universal-gateway/overview.mdx

```diff
@@ -1,58 +1,40 @@
 ---
 title: Universal Gateway Overview
 sidebarTitle: Overview
+description: Learn about ngrok's suite of common tools for building API and device gateways, identity-aware proxies, and site-to-site connectivity.
 ---
 
 ngrok's Universal Gateway is a suite of common tools for building API and device gateways, identity-aware proxies, and site-to-site connectivity.
 It secures, accelerates, and protects your applications.
-We call it a Universal Gateway because it supports [TCP](/universal-gateway/tcp/) and [TLS](/universal-gateway/tls/) applications as well as [HTTP/S](/universal-gateway/http/).
 
-## Concepts
+**What makes it "universal"?**
 
-Dive into the different parts of the Universal Gateway like Endpoints, Bindings, Pools, and Domains to understand how they work.
+The Universal Gateway can be used with software running locally or in the cloud, and with devices running on-premises or distributed in the field.
+It supports [TCP](/universal-gateway/tcp/) and [TLS](/universal-gateway/tls/) applications as well as [HTTP/S](/universal-gateway/http/).
+You can deliver traffic to internal and public APIs as well as orchestrate traffic across your devices.
+It's globally distributed by default and provides support for multiple environments with minimal configuration.
 
-Learn more about Universal Gateway Concepts:
+## Features
 
-- [Domains →](/universal-gateway/domains/)
-- [TCP Addresses →](/universal-gateway/tcp-addresses/)
-- [TLS Certificates →](/universal-gateway/tls-certificates/)
-- [Edges →](/universal-gateway/edges/)
+Here are some of the key features of the Universal Gateway platform:
 
-## Endpoints
+- [**Traffic Policy**](/traffic-policy/): filter, match, manage, and orchestrate traffic to your endpoints.
+- [**Traffic observability**](/obs/): capture request and response data; replay requests against your endpoints for debugging replication.
+- [**Identity and access management**](/iam/): manage credentials for human users and automated processes.
+- [**Kubernetes support**](/k8s/): ngrok's Kubernetes Operator supports ingress resources as well as cross-platform Gateway API configuration resources.
 
-[Endpoints](/universal-gateway/endpoints/) are how ngrok enables traffic to reach your services.
+## Use cases
 
-Learn more about Endpoints:
+Here are some of the most common use cases for the Universal Gateway, from the [examples collection](/universal-gateway/examples/):
 
```

Comment:
Might be worth converting this list into a grid of cards, just to break up the visual presentation on the page a bit since we have so many bulleted lists here already. Just a suggestion

https://www.mintlify.com/docs/components/cards#cards